taprpc/taprootassets.proto

```diff
@@ -348,6 +348,25 @@ message GenesisReveal {
     GenesisInfo genesis_base_reveal = 1;
 }
 
+message DecimalDisplayOpt {
+    // This value signals that the decimal display value has been set.
+    bool is_set = 1;
```

Comment:
Looking at the RPC code in the next commit, wouldn't the whole `decimal_display` message just be `null` if it wasn't set? So the boolean is kind of superfluous as it would always be `true` (if the parent message is there)?

Reply:
I thought `null` did not exist in the proto world? it will instead set the default values so `is_set` will be `false`

Reply:
It does, but only for messages, not for primitive types.